Libraries/react-native/react-native.js

```diff
@@ -121,6 +121,7 @@ const ReactNative = {
   get ColorPropType() { return require('ColorPropType'); },
   get EdgeInsetsPropType() { return require('EdgeInsetsPropType'); },
   get PointPropType() { return require('PointPropType'); },
+  get IndicatorSizePropType() { return require('IndicatorSizePropType'); },
```

Comment:
I don't think we need to export that prop type as it's only used by one component.